aggsender/types/multisig_committee.go

```diff
@@ -91,6 +91,14 @@ func (m *MultisigCommittee) Threshold() *big.Int {
 	return m.threshold
 }
 
+// ThresholdInt returns the signature threshold as an int64.
+func (m *MultisigCommittee) ThresholdInt() (int64, error) {
+	if m.threshold.IsInt64() {
+		return m.threshold.Int64(), nil
+	}
+	return 0, fmt.Errorf("threshold %s doesn't fit in int64", m.threshold.String())
+}
```

Comment:
Maybe makes slightly more sense to represent it as `uint64` (since it shouldn't be negative), rename it and remove the original `Threshold` function. WDYT?

```suggestion
// Threshold returns the signature threshold required for quorum.
func (m *MultisigCommittee) Threshold() (uint64, error) {
	if m.threshold.IsUint64() {
		return m.threshold.Uint64(), nil
	}
	return 0, fmt.Errorf("threshold %d doesn't fit in uint64", m.threshold)
}
```

Btw maybe it makes sense to discuss with protocol team to change it on their end as well to int64/uint64 (if it is not already late, because of ongoing audit etc.)?

Reply:
The type in the [contract](https://github.com/agglayer/agglayer-contracts/blob/e2a68464f2f34a759731089ed26d2769fce1c2d3/contracts/v2/lib/AggchainBase.sol#L77) is `uint256`, so I agree that we can use `uint64`